src/rust/pt-br.modescanlator/res/source.json

```
@@ -0,0 +1,24 @@
+{
+	"info": {
+		"id": "pt-br.modescanlator",
+		"lang": "pt-br",
+		"name": "Mode Scanlator",
+		"version": 1,
+		"url": "https://site.modescanlator.net",
+		"nsfw": 2
```

is there nsfw content?
```suggestion
		"nsfw": 0
```

---

No.